examples/material_gallery/lib/demo/tooltip_demo.dart

```diff
@@ -19,8 +19,7 @@ class TooltipDemo extends StatelessWidget {
       ),
       body: new Builder(
         builder: (BuildContext context) {
-          return new Column(
-            crossAxisAlignment: CrossAxisAlignment.stretch,
```

Comment:
Does CrossAxisAlignment.stretch want a dartdoc to say that if you're using this you might want block, or am I reading too much into this?